backend/src/main/java/gov/cdc/usds/simplereport/service/PatientBulkUploadServiceAsync.java

```diff
@@ -116,30 +116,35 @@ public CompletableFuture<Set<Person>> savePatients(byte[] content, UUID facility
 
         // create new person with current organization, then add to new patients list
         Person newPatient =
-            new Person(
-                currentOrganization,
-                assignedFacility.orElse(null),
-                null, // lookupid
-                extractedData.getFirstName().getValue(),
-                extractedData.getMiddleName().getValue(),
-                extractedData.getLastName().getValue(),
-                extractedData.getSuffix().getValue(),
-                parseUserShortDate(extractedData.getDateOfBirth().getValue()),
-                address,
-                country,
-                parsePersonRole(extractedData.getRole().getValue(), false),
-                extractedData.getEmail().getValue() == null
-                    ? Collections.emptyList()
-                    : List.of(extractedData.getEmail().getValue()),
-                convertRaceToDatabaseValue(extractedData.getRace().getValue()),
-                convertEthnicityToDatabaseValue(extractedData.getEthnicity().getValue()),
-                null, // tribalAffiliation
-                convertSexToDatabaseValue(extractedData.getBiologicalSex().getValue()),
-                parseYesNoUnk(extractedData.getResidentCongregateSetting().getValue()),
-                parseYesNoUnk(extractedData.getEmployedInHealthcare().getValue()),
-                null, // preferredLanguage
-                null // testResultDeliveryPreference
-                );
+            Person.builder()
```

Comment:
it was super annoying to deal with the multi-param constructor off the person object so I added a new one with the Builder annotation to make it more manageable.